assessment_store/tasks/db_tasks.py

```diff
@@ -135,6 +133,7 @@ def seed_local_assessment_store_db(c):
             # Insert scoring systems
             one_to_five_id = str(uuid.uuid4())  # Generate a UUID for OneToFive
             zero_to_three_id = str(uuid.uuid4())  # Generate a UUID for OneToThree
+            zero_to_one_id = str(uuid.uuid4())  # Generate a UUID for ZeroToOne
```

Comment:
Are you using this `ZeroToOne` anywhere?

Reply:
currently using the default 1-5 for these rounds until we get the confirmation from the onboarding team.